submissions/.gitignore

```diff
@@ -1,5 +0,0 @@
-# Ignore everything in this directory
-*
-
-!.gitignore
-.DS_Store
```

Comment:
@Dufanaakinloluwa Please revert changes to this file, we need the lines removed to ensure that .gitignore is included and[ .DS_Store](https://www.wikiwand.com/en/.DS_Store) is untracked.

Reply:
please how do i do that